v3/src/models/formula/formula-manager.ts

```diff
@@ -133,24 +136,25 @@ export class FormulaManager {
     adapter.recalculateFormula(formulaContext, extraMetadata, casesToRecalculate)
   }
 
-  getAllFormulas() {
-    return this.adapters.flatMap(a => a.getAllFormulas())
+  getActiveFormulas() {
+    return this.adapters.flatMap(a => a.getActiveFormulas())
   }
 
-  registerAllFormulas() {
+  registerActiveFormulas() {
     reaction(() => {
       // Observe all the formulas
-      return this.getAllFormulas().map(({ formula, extraMetadata }) => (
+      return this.getActiveFormulas().map(({ formula, extraMetadata }) => (
         { id: formula.id, formula: formula.display, extraMetadata }
       ))
     }, () => {
-      this.unregisterDeletedFormulas()
+      const activeFormulas: Set<string> = new Set()
```

Comment:
FYI, I think you can just use:
```suggestion
      const activeFormulas = new Set<string>()
```